src/common/requests.js

```diff
@@ -7,7 +9,9 @@ function request(method, url, headers, urlParams = {}, body) {
       : ''
   }`, {
     method,
-    headers: new Headers(headers),
+    headers: new Headers(Object.assign(headers, {
```

Comment:
Some requests like login should not pass Authorization header.  If secure == false do not send authorization header. By default make it true.

Reply:
And what exactly 'secure' is? Should i add a new parameter to function request or 'secure' is header's field?